core/trino-main/src/main/java/io/trino/connector/informationschema/InformationSchemaMetadata.java

```diff
@@ -338,8 +338,8 @@ private <T> Optional<Set<String>> filterString(TupleDomain<T> constraint, T colu
         if (domain.isSingleValue()) {
             return Optional.of(ImmutableSet.of(((Slice) domain.getSingleValue()).toStringUtf8()));
         }
-        if (domain.getValues() instanceof EquatableValueSet) {
-            Collection<Object> values = ((EquatableValueSet) domain.getValues()).getValues();
+        if (domain.getValues() instanceof EquatableValueSet equatableValueSet) {
```

Comment:
Use a more concise name such as `set`. It makes the code easier to read.

Same comment for the ones below, where appropriate.